locale/en/locale.cfg

```diff
@@ -27,31 +55,89 @@ warehouse-active-provider=Active Provider Warehouse
+storehouse-requester-small=Small Requester Storehouse
+storehouse-buffer-small=Small Buffer Storehouse
+
+storehouse-basic-tiny=Tiny Storehouse
+storehouse-passive-provider-tiny=Tiny Passive Provider Storehouse
+storehouse-storage-tiny=Tiny Storage Storehouse
+storehouse-active-provider-tiny=Tiny Active Provider Storehouse
+storehouse-requester-tiny=Tiny Requester Storehouse
+storehouse-buffer-tiny=Tiny Buffer Storehouse
+
 [item-description]
-warehouse-basic=Stores many many items
-warehouse-passive-provider=Passive Provider Warehouse
-warehouse-storage=Storage Warehouse
-warehouse-active-provider=Active Provider Warehouse
-warehouse-requester=Requester Warehouse
-warehouse-buffer=Buffer Warehouse
+warehouse-basic=Great storage capacity
+warehouse-passive-provider=Great storage capacity
+warehouse-storage=Great storage capacity
+warehouse-active-provider=Great storage capacity
+warehouse-requester=Great storage capacity
+warehouse-buffer=Great storage capacity
 
-storehouse-basic=Stores some items
-storehouse-passive-provider=Passive Provider Storehouse
-storehouse-storage=Storage Storehouse
-storehouse-active-provider=Active Provider Storehouse
-storehouse-requester=Requester Storehouse
-storehouse-buffer=Buffer Storehouse
+warehouse-basic-small=Great storage capacity
+warehouse-passive-provider-small=Great storage capacity
+warehouse-storage-small=Great storage capacity
+warehouse-active-provider-small=Great storage capacity
+warehouse-requester-small=Great storage capacity
+warehouse-buffer-small=Great storage capacity
+
+warehouse-basic-tiny=Great storage capacity
+warehouse-passive-provider-tiny=Great storage capacity
+warehouse-storage-tiny=Great storage capacity
+warehouse-active-provider-tiny=Great storage capacity
+warehouse-requester-tiny=Great storage capacity
+warehouse-buffer-tiny=Great storage capacity
+
+storehouse-basic=Good storage capacity
+storehouse-passive-provider=Good storage capacity
+storehouse-storage=Good storage capacity
+storehouse-active-provider=Good storage capacity
+storehouse-requester=Good storage capacity
+storehouse-buffer=Good storage capacity
+
+storehouse-basic-small=Good storage capacity
+storehouse-passive-provider-small=Good storage capacity
+storehouse-storage-small=Good storage capacity
+storehouse-active-provider-small=Good storage capacity
+storehouse-requester-small=Good storage capacity
+storehouse-buffer-small=Good storage capacity
+
+storehouse-basic-tiny=Good storage capacity
+storehouse-passive-provider-tiny=Good storage capacity
+storehouse-storage-tiny=Good storage capacity
+storehouse-active-provider-tiny=Good storage capacity
+storehouse-requester-tiny=Good storage capacity
+storehouse-buffer-tiny=Good storage capacity
 
 [technology-name]
 warehouse-research=Warehousing
 warehouse-logistics-research=Logistics Warehousing
+warehouse-research-tiny=Tiny Warehousing
+warehouse-logistics-research-tiny=Tiny Logistics Warehousing
```

Comment:
Feeling like just adding "Tiny" to the existing strings to get the new versions is less interesting than it could be. I had the idea of "Warehouse Compression" or "Warehouse Efficiency"—and another, but I forgot it between skimming the locale files during rebasing your original branch (before the PR) and now. 😞

Reply:
Oooooo I like the sound of "Warehouse Compression"